code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -432,7 +433,8 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			wearer.dropItemToGround(equipped_item, force = TRUE)
 
 /datum/species/proc/update_no_equip_flags(mob/living/carbon/wearer, new_flags)
-	no_equip_flags = new_flags
+	if(new_flags)
+		no_equip_flags = new_flags
```

Comment:
This means you can't update a species equip flags to `NONE`?

Reply:
You shouldn't be anyways, I intentionally left out belt/pockets/ID since it should be tied to the jumpsuit instead.

Reply:
ah fuck you're right i was thinking backwards